packages/components/src/styles/includes/index.scss

```diff
@@ -225,47 +225,6 @@
   }
 }
 
-@mixin close-button(
-  $size: "var(--calcite-internal-close-size, 1.5rem /* 24px */)",
-  $padding: "0",
-  $color: "var(--calcite-close-icon-color, var(--calcite-color-text-1))"
-) {
-  .close {
-    @apply border-none
-      cursor-pointer
-      focus-base
-      items-center
-      m-0
-      rounded-half
-      transition-default;
-
-    background-color: var(--calcite-close-background-color, var(--calcite-color-transparent));
-    -webkit-appearance: none;
-    display: flex;
-    align-content: center;
-    justify-content: center;
-    color: #{$color};
-    block-size: #{$size};
-    inline-size: #{$size};
-    padding: #{$padding};
-
-    &:hover,
-    &:focus {
-      background-color: var(--calcite-close-background-color-hover, var(--calcite-color-transparent-hover));
-    }
-    &:focus {
-      @apply focus-inset;
-    }
-    &:active {
-      background-color: var(--calcite-close-background-color-press, var(--calcite-color-transparent-press));
-    }
-
-    calcite-icon {
-      color: inherit;
-    }
-  }
-}
-
```

Comment:
This Sass mixin was only being used by Chip, and all of this is accounted for now that its using an action as the close button.